frontend/views/containers/Overview.vue

```diff
@@ -1,139 +1,110 @@
 <template lang='pug'>
 div
-  p.has-text-1 Group members and their pledges
-  //- input(type="text" v-model='ratioWidthPadding')
-  .c-chart-wrapper
-    svg.c-chart(
-      width='100%'
-      :viewBox='`0 0 526 ${ratioY}`'
-      preserveAspectRatio='xMidYMid meet'
-      aria-labelledby='title'
-      role='img'
-    )
-      g.bars(
-        v-for='(member, index) in fakeData.members'
-        :transform='position(index)'
-      )
-        rect(
-          :width='width'
-          :height='height(member.delta)'
-          :y='positionY(member.delta)'
-          :fill='color(member.delta)'
-        )
-
-        rect(
-          v-if='member.delta >= 0 && (ratioY / 2 - height(member.delta)) < surplusPosition'
-          :width='width'
-          :height='surplusPosition'
-          :y='positionY(member.delta)'
-          fill='#A0D10E'
-        )
-
-      g.line(:transform='`translate(0,${ratioY/2})`')
-        line(x1='0' y1='0' :x2='maxGraphWidth' y2='0' stroke='#dbdbdb' stroke-width='1')
-        foreignObject(x='469' y='-9' width='57' height='18')
-          .tag.mincome Mincome
-
-      g.line(:transform='`translate(0,${surplusPosition})`')
-        line(x1='0' y1='0' :x2='maxGraphWidth' y2='0' stroke='#dbdbdb' stroke-width='1' stroke-dasharray='1')
-        foreignObject(x='469' y='-9' width='50' height='18')
-          .tag Surplus
-
-    .c-chart-info
-      i18n.label(tag='label') Total amount needed
-        .c-total-amount $750
-
-      label.label
-        i18n Total pledged
-        .c-total-pledge $800
-
-      label.label
-        i18n Surplus
-        .c-surplus $50
-
-      i18n.help(tag='p') This amount will not be used
+  i18n.has-text-1(tag='p') Group members and their pledges
+  .c-chart
+    ul.c-chart-legends
+      graph-legend-item.c-wlegend-inline(
+        :amount='withCurrency(this.totalNeeded)'
+        color='warning-solid'
+      ) {{ L('Total needed') }}
```

Comment:
<img width="774" alt="Screen Shot 2020-01-08 at 5 42 30 PM" src="https://user-images.githubusercontent.com/138706/72030579-4b3ef600-323e-11ea-8870-52237d67f12f.png">

Noticing two things here:

1. I find the phrasing "Total needed" and "Needed" to be confusing. The orange area represents not the amount needed, but the total income members say they already have. So I recommend using a label like "Existing income" (cc @mmbotelho)
2. For some reason the red box next to "Needed" has a black outline. I think it would look better if that were removed, so that it's consistent with the other legend items